storage/innobase/xtrabackup/test/suites/keyring/pxb-1942.sh

```diff
@@ -1,13 +1,16 @@
 #
 # PXB-1942: Backup fails when database is on a fast storage with redo archive logs enabled
 #
+. inc/common.sh
+. inc/keyring_file.sh
 
 require_server_version_higher_than 8.0.16
 
 mkdir $TEST_VAR_ROOT/b
 
 start_server --innodb-redo-log-archive-dirs=":$TEST_VAR_ROOT/b"
 
+mysql -e "set global innodb_redo_log_encrypt=on"
```

Comment:
Please add a testcase that tests without redo log encryption

Reply:
added.